ha/core/controllers/pcs/node_controller.py

```diff
@@ -112,20 +112,24 @@ def standby(self, nodeid: str) -> dict:
         node_status = self.nodes_status([nodeid]).get(nodeid)
         Log.info(f"Current {nodeid} status is {node_status}")
         if node_status == NODE_STATUSES.STANDBY.value:
-            status = f"Node {nodeid} is already running in standby mode."
-        elif node_status != NODE_STATUSES.ONLINE.value:
-            return {"status": const.STATUSES.FAILED.value, "output": "",
-                    "error": f"Failed to put node in standby as node is in {node_status}"}
-        else:
+            return {"status": const.STATUSES.SUCCEEDED.value, "output":
+                f"Node {nodeid} is already running in standby mode.", "error": ""}
+        elif node_status == NODE_STATUSES.STANDBY_WITH_RESOURCES_RUNNING.value:
```

Comment:
There could be other node_status values that we need to handle here e.g. MAINTENANCE. Why are we changing this check?

Reply:
Got it, we have else below.